cloudevents/config/config-defaults.yaml

```diff
@@ -21,6 +21,7 @@ metadata:
     app.kubernetes.io/instance: default
     app.kubernetes.io/part-of: tekton-cloudevents
 data:
+  default-cloud-events-sink: http://sockeye.default.10.108.51.172.nip.io/
```

Comment:
Please let's not add this default in the repo 🙏

Reply:
oh my bad, I forgot I added this, I'l remove it :bow:

Reply:
fixed